README.md

```diff
@@ -93,6 +93,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The amount of time before a user's "Basic" authentication
                 // token expires.
                 "timeout_in_seconds": 3600
+            },
+
+            // Defines required OIDC related configuration
+            "oicd": {
+                // The ip address or hostname of the OIDC provider
+                "config_host": "oidc.example.com",
+
+                // The port the OIDC provider is listening on
+                "port": "8080",
+
+                // The path to the OIDC well-known endpoint
+                "uri": "/realms/example/.well-known/openid-configuration",
```

Comment:
We could use `well_known_uri`, additionally, we could drop the `.well-known/openid-configuration` part since that is well known.

We could add something like the following:
> Retrieves configuration of OIDC server to contact the appropriate end-points.

Reply:
Yeah, let's drop the parts that aren't required (especially for things that can be derived).

> We could add something like the following:
>
>> Retrieves configuration of OIDC server to contact the appropriate end-points.

Try to use phrasing that matches that of other properties. Overall, that wording seems fine. We'll work on it.